### PR TITLE
OAuth V2 documentation added

### PR DESCRIPTION
**Pull request by Ankush1994**

### CODE REVIEW

File: docs/books/authenticationv2.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+id: authenticationv2
+title: Authentication Service V2
+sidebar_label: Authentication V2
+---
+
+A token is a piece of data that has no meaning or use on its own, but combined with the correct tokenization system, becomes a vital player in securing your application. Token based authentication works by ensuring that each request to a server is accompanied by a signed toke (RFC 7519) that defines a compact and self-contained method for securely transmitting information between parties encoded as a JSON object.
```
Comment (archit-p) [inline docs/books/authenticationv2.md:7]: Typo in `signed toke` -> `signed token`

File: docs/books/authenticationv2.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+id: authenticationv2
+title: Authentication Service V2
+sidebar_label: Authentication V2
+---
+
+A token is a piece of data that has no meaning or use on its own, but combined with the correct tokenization system, becomes a vital player in securing your application. Token based authentication works by ensuring that each request to a server is accompanied by a signed toke (RFC 7519) that defines a compact and self-contained method for securely transmitting information between parties encoded as a JSON object. 
+
+The Authentication V2 API enables you to manage all aspects of user identity when you use Auth0. It offers endpoints so your users can log in, sign up, log out, access APIs, and more.
```
Comment (archit-p) [inline docs/books/authenticationv2.md:9]: Auth0 is a company that provides JWT auth, I don't think it was intended to have them mentioned here

File: docs/books/oauthv2.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+##### Sample OAuth Link
+
+```
+https://oauth-staging.deskera.xyz/?client_id=cl_pub_1234&scope=write&state=65921&response_type=code
+```
+
+#### Step 2: Users connect their Deskera account
+
+Once on the Deskera App auth page, the user would be prompted to input his/her Deskera user name and password, select the Organization we would like to connect and asked to authorize the app. The user should be already registered to Deskera Books and have atleast one organisation before coming to this step.
```
Comment (archit-p) [inline docs/books/oauthv2.md:37]: We don't prompt user to enter user name and password anymore

File: docs/books/oauthv2.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+#### Step 3: Redirection request with Auth Code
+
+After the user connects their existing account to your platform, we'll make an API call to the `redirect_uri` (provided at the time of partner registration) with the below parameters
```
Comment (archit-p) [inline docs/books/oauthv2.md:41]: We are redirecting now, so no API call

**Review by archit-p — CHANGES_REQUESTED** [review]

File: docs/books/oauthv2.md (hunk 4 above)
Comment (Ankush1994) [reply docs/books/oauthv2.md:41]: Done

File: docs/books/oauthv2.md (hunk 3 above)
Comment (Ankush1994) [reply docs/books/oauthv2.md:37]: removed

File: docs/books/authenticationv2.md (hunk 2 above)
Comment (Ankush1994) [reply docs/books/authenticationv2.md:9]: Removed reference to Auth0

File: docs/books/authenticationv2.md (hunk 1 above)
Comment (Ankush1994) [reply docs/books/authenticationv2.md:7]: Fixed

**Pull request merged**